kogito-editors-js/packages/boxed-expression-component/showcase/package.json

```diff
@@ -4,16 +4,17 @@
   "license": "Apache-2.0",
   "private": true,
   "dependencies": {
+    "customize-cra": "^1.0.0",
     "react": "^17.0.2",
+    "react-app-rewired": "^2.1.8",
     "react-dom": "^17.0.2",
-    "react-scripts": "^4.0.3",
-    "customize-cra": "^1.0.0",
-    "react-app-rewired": "^2.1.8"
+    "react-scripts": "^4.0.3"
   },
   "devDependencies": {
     "@types/node": "^15.0.2",
     "@types/react": "^17.0.5",
     "@types/react-dom": "^17.0.3",
+    "cypress": "^8.1.0",
```

Comment:
May we add this line in the `kogito-editors-js/package.json` as well?

Reply:
I think yes, however not sure, if then it will be needed in `showcase`, aren't the parent dependencies available in childs?

Reply:
@jomarko I believe it would be nice to have it on both places. Here we're declaring dependencies for the `showcase` project. And `kogito-editors-js/package.json` help us to keep all dependencies centralized in a single place (so, all projects may share the same versions).

Reply:
@karreiro changes are in